ipfshttpclient/client/__init__.py

```diff
@@ -14,9 +14,9 @@
 DEFAULT_ADDR = multiaddr.Multiaddr(os.environ.get("PY_IPFS_HTTP_CLIENT_DEFAULT_ADDR", '/dns/localhost/tcp/5001/http'))
 DEFAULT_BASE = str(os.environ.get("PY_IPFS_HTTP_CLIENT_DEFAULT_BASE", 'api/v0'))
 
-VERSION_MINIMUM   = "0.4.23"
+VERSION_MINIMUM   = "0.5.0"
 VERSION_BLACKLIST = []
-VERSION_MAXIMUM   = "0.8.0"
+VERSION_MAXIMUM   = "0.9.0"
```

Comment:
Why is this `0.9.0` and not `0.8.0` ?

Don't we want the warning behavior when connecting to `0.9.0`?

Reply:
The check is `VERSION_MINIMUM <= version < VERSION_MAXIMUM` since it couldn't otherwise cover the entire version range of the maximum supported version series. (Think of it like Python's `range` function. :wink:)

Reply:
I'd add a comment about it being an inclusive lower bound and exclusive upper bound.